tests/fixtures/siteconfig.json5

```diff
@@ -12,5 +12,6 @@
     	resizable: true,
     	alwaysOnTop: true
     },
-    surveyTriggersUrl: "http://localhost:8083/surveys/triggers.json"
+    surveyTriggersUrl: "http://localhost:8083/surveys/triggers.json",
+    defaultValues: "gpii\\defaultSettings.json5"
```

Comment:
This name `defaultValues` is confusing. It might be more reasonable using a name like "defaultSettingsFileLocation".